finder/tagged_test.go

```diff
@@ -74,34 +75,139 @@ func TestParseSeriesByTag(t *testing.T) {
 	assert := assert.New(t)
 
 	ok := func(query string, expected []TaggedTerm) {
-		p, err := ParseSeriesByTag(query)
+		p, err := ParseSeriesByTag(query, nil)
 		assert.NoError(err)
-		assert.Equal(expected, p)
+		length := len(expected)
+		if length < len(p) {
```

Comment:
In what case len(expected) can be not equal to len(p)? As it seems that previously it was a valid condition and test should anyway end up with an error.

Reply:
ParseSeriesByTag return a slice (p). If a bug, length can be non equal of length of expected results. `assert.Equal` produce not readable result for compare slices or map. But not required and can be removed.

Reply:
Yeah I understand that, but still that seems to be a good idea to produce an error here (maybe custom one if assert gives non-readable message).